app/views/invoices/_form.html.haml

```diff
@@ -0,0 +1,74 @@
+= form_for @invoice do |f|
+  .row
+    .col-md-6
+      .box.box-primary
+        .box-body
+          .form-group
+            = label_tag :client
+            = f.select(:client_id, get_clients, {}, class: "form-control")
+          .form-group
+            = label_tag :address
+            = label_tag "","F1/F2, JBR Arcade, Science City", class: "form-control remove_border_element"
+          .form-group
+            = label_tag :currency, "Currency"
+            = currency_select("client", "currency_code", { selected: "inr"}, {}, class: 'form-control')
+    .col-md-6
+      .box.box-primary
+        .box-body
+          .form-group
+            = f.label :invoice_number
+            = f.text_field :invoice_number, class: "form-control"
+          .form-group
+            = f.label :date_of_issue
+            = f.text_field :date_of_issue, {data: {provide: :datepicker}, class: "form-control"}
+          .form-group
+            = f.label :due_date
+            = f.text_field :paid_to_date, {data: {provide: :datepicker}, class: "form-control"}
+          .form-group
+            = f.label :po_number
+            = f.text_field :po_number, class: "form-control"
+  %h3 Line Items
+  .row
+    .col-md-12
+      .box
+        .box-header
+        .box-body.table-responsive.no-padding
+          %table.table.table-hover
+            %tbody
+              %tr
+                %th.width20 Item
+                %th.width30 Description
+                %th.width10 Unit Cost
+                %th.width10 Qty
+                %th.width10 Line Total
+                %th.width20 Remove
+          #line_items
+            = f.fields_for :line_items do |line_item|
+              = render 'line_item_fields', f: line_item
+        .links
+          = link_to_add_association 'add line item', f, :line_items
+  .row
+    .col-md-7
+    .col-md-5
+      .box.remove_border_element
+        .box-body
+          .form-group
+            = label_tag "Invoice Total(INR) : "
```

Comment:
No Sir, for static layout I have put INR, for dynamic data, we will calculate from currency